src/components/v2/V2Project/TreasuryStats.tsx

```diff
@@ -77,21 +77,22 @@ export default function TreasuryStats() {
           </div>
         }
         style={{ marginBottom: spacing }}
+        loading={!Boolean(balanceInDistributionLimitCurrency)}
       />
 
-      {distributionLimit?.gt(0) ? (
-        <StatLine
-          statLabel={<Trans>Distributed</Trans>}
-          statLabelTip={
-            <Trans>
-              The amount that has been distributed from the Juicebox balance in
-              this funding cycle, out of the current funding target. No more
-              than the funding target can be distributed in a single funding
-              cycle—any remaining ETH in Juicebox is overflow, until the next
-              cycle begins.
-            </Trans>
-          }
-          statValue={
+      <StatLine
+        loading={!Boolean(distributionLimit)}
+        statLabel={<Trans>Distributed</Trans>}
+        statLabelTip={
+          <Trans>
+            The amount that has been distributed from the Juicebox balance in
+            this funding cycle, out of the current funding target. No more than
+            the funding target can be distributed in a single funding cycle—any
```

Comment:
'cycle—any' -> 'cycle — any' ?

Reply:
Funnily enough, looks like this is the proper use: https://www.thesaurus.com/e/grammar/em-dash/

I didn't change this though, was already there.